Add a docs page for OTEL_EXPORTER_OTLP_ENDPOINT  codemod (#700)
hasura/ddn-docs: docs/project-configuration/upgrading-project-config/fix-traces-env-var.mdx

---

> sidebar_position: 4

Reviewer: ```suggestion
sidebar_position: 5
```

sidebar_label: Fix environment variable for OTLP endpoint
description: "Fix the name of the environment variable used to configure OTLP endpoint"
keywords:
  - hasura ddn
  - data delivery network
  - hasura cli
  - project configuration
  - hasura metadata management
  - yaml configuration
  - api development
  - hasura build profiles
  - hasura subgraphs
  - hasura project management
seoFrontMatterUpdated: true
toc_max_heading_level: 4
---

# Fix the OTLP endpoint environment variable

## What has happened?

> In previous versions of the DDN CLI, the environment variable `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT`

Reviewer: ```suggestion
In DDN CLI versions **v2.7.0** and below, the environment variable `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT`
```

---
sidebar_position: 5

> sidebar_label: Fix environment variable for OTLP endpoint

Reviewer 2: ```suggestion
sidebar_label: Fix env var for OTLP endpoint
```
